api/proto/dop/issue/core/core.proto

```diff
@@ -200,6 +200,16 @@ service IssueCoreService {
     };
   };
 
+  rpc CleanOrgCustomFileds(CleanOrgCustomFiledsRequest)
+  returns (CleanOrgCustomFiledsResponse) {
+    option (google.api.http) = {
+      post: "/api/issues/actions/clear-org-customfileds",
+    };
+    option (erda.common.openapi) = {
```

Comment:
It's unnecessary to expose it as an OpenAPI.

Reply:
ok